introduction/getting-started.md

```diff
@@ -1,42 +1,42 @@
 # Getting started
 
-Let us give you a quick overview of how the proxy is used. In this section, we give you a quick introduction to the Dashboard and we'll also set up a simple project.
+Let us give you a quick overview of how the proxy is used. In this section, we give you a quick introduction to the Dashboard 2.0 and we'll also set up a simple project.
 
 ## Registration & Login
 
 To use Easyling, you need to register and set up an account for the service at [https://app.easyling.com](https://app.easyling.com). You can start using the service right away after registration.
 
-Logging in, you will be taken to the Dashboard, which is the project management center, every detail of which we'll get to in later sections of this manual. There will be no existing projects at the outset, so let's try setting one up.
+Logging in, you will be taken to the Dashboard 2.0, which is the new and improved project management center, every detail of which we'll get to in later sections of this manual. There will be no existing projects at the outset, so let's try setting one up.
 
 ## Setting up Your First Project
 
 To do this, click on **Create new project** dropdown at the top and choose **Add project**.
 
-This opens the **Add project** dialog box, where you can enter the URL of the website you would like to translate, and also select the website language; this sets the source language of the translation project. Click on Advanced Settings to access extended functionality, such as custom SRX files.
+This opens the **Create new project** dialog box, where you can enter the URL of the website you would like to translate and select the website language. You can also add an alias to your project. This is an easy to remember name that you can recognise your project from. You can also decide to start a Discovery immediately after creating your project to skip the next section. <br /> Click on Advanced Settings to access extended functionality, such as custom SRX files.
 
-![Add Project](/img/dashboard/add_project_dialog.png)
+![Add Project](/img/dashboard2/add_project_dialog.png)
 
-### Add a Target Lanuage
+## Running a Discovery
 
-You will also need to add your target language(s), so use the option on the Dashboard to add then to the project. It's not just that there is not much to do in terms of translation without a target: many crucial features (including the Preview proxy and the entire Workbench) are entirely unavailable as long as no target languages are set.
+![Crawl wizard](/img/dashboard2/crawl_step_5.png)
```

Comment:
Maybe move this a paragraph down? It looks jarring on this screen to have an image right after the headline